### PR TITLE
Uncap the maximum framerate for high refresh monitors. Fixes #1796

### LINKED ISSUE #1796: Increase FPS cap from 100 to 240
Why not? If someone has the monitor to run that, then let them!

### PR DESCRIPTION
**Pull request by aeoncleanse**

### CODE REVIEW

File: lua/ui/game/gamemain.lua [hunk 1]
```diff
@@ -281,7 +281,7 @@ function AdjustFrameRate()
         local data = utils.StringSplit(options.primary_adapter, ',')
         local hz = tonumber(data[3])
         if hz then
-            fps = math.max(60, math.min(100, hz))
```
Comment (duk3luk3) [inline lua/ui/game/gamemain.lua]: Are you sure SC_FrameTimeClamp supports values > 100?

Comment (aeoncleanse) [inline lua/ui/game/gamemain.lua]: I know the game supports non-frameclamped numbers. I've seen SC running at >300fps.,.

Comment (duk3luk3) [inline lua/ui/game/gamemain.lua]: OK. We'll see then :-)

Comment (aeoncleanse) [inline lua/ui/game/gamemain.lua]: OK well, someone definitely did it, and showed me a video... But now when I run sc_FrameTimeClamp -1 (To uncap it) it just goes to 100... Trouble is, my screen is exactly a 100Hz display...

**Pull request merged**